sermant-agentcore/sermant-agentcore-core/src/main/java/io/sermant/core/utils/LabelGroupUtils.java

```diff
@@ -43,26 +38,6 @@ public class LabelGroupUtils {
 
     private static final String KV_SEPARATOR = "=";
 
-    /**
-     * The kv separator used in the query
-     */
-    private static final String LABEL_QUERY_SEPARATOR = ":";
-
-    /**
-     * Query label prefix
-     */
-    private static final String LABEL_PREFIX = "label=";
-
-    /**
-     * Key-value pair length
-     */
-    private static final int KV_LEN = 2;
-
-    /**
-     * Default group
-     */
-    private static final String DEFAULT_GROUP_KEY = "GROUP";
-
     private LabelGroupUtils() {
```

Comment:
move remained method createLabelGroup to AbstractGroupConfigSubscriber for its usage

Reply:
done